v1.1-beta/matrixx-digital-commerce/matrixx-tmf-usage-consumption-management/cnf-testsuite-results-20230806-153101-592.yml

```diff
@@ -0,0 +1,232 @@
+---
+name: cnf testsuite
+testsuite_version: v0.42.3
+status:
+command: /home/dwilmes/.mtx/konstruxx/working/tests/testTMFUsageConsumptionManagement/cnf-testsuite
```

Comment:
@deerowbear @daniel-wilmes generic please for end-users to be able to repeat testing if desired